contracts/feature-tests/use-module/tests/gov_module_whitebox_test.rs

```diff
@@ -0,0 +1,603 @@
+        |sc| {
+            sc.vote(proposal_id, VoteType::UpVote);
+        },
+    );
+
+    // queue ok
+    current_block_nonce = 45;
+    world.set_state_step(SetStateStep::new().block_nonce(current_block_nonce));
+    world.whitebox_call(
+        &use_module_whitebox,
+        ScCallStep::new().from(FIRST_USER_ADDRESS_EXPR).no_expect(),
+        |sc| {
+            sc.queue(proposal_id);
+        },
+    );
+
+    // try execute too early
+    world.whitebox_call_check(
+        &use_module_whitebox,
+        ScCallStep::new().from(FIRST_USER_ADDRESS_EXPR).no_expect(),
+        |sc| {
+            sc.execute(proposal_id);
+        },
+        |r| {
+            r.assert_user_error("Proposal is in timelock status. Try again later");
+        },
+    );
+
+    // execute ok
+    current_block_nonce += LOCKING_PERIOD_BLOCKS;
+    world.set_state_step(SetStateStep::new().block_nonce(current_block_nonce));
+    world.whitebox_call(
+        &use_module_whitebox,
+        ScCallStep::new().from(FIRST_USER_ADDRESS_EXPR).no_expect(),
+        |sc| {
+            sc.execute(proposal_id);
+        },
+    );
+
+    // after execution, quorum changed from 1_500 to the proposed 1_000
+    world.whitebox_query(&use_module_whitebox, |sc| {
+        assert_eq!(sc.quorum().get(), managed_biguint!(1_000));
+        assert!(sc.proposals().item_is_empty(1));
+    });
+
+    world.check_state_step(CheckStateStep::new().put_account(
+        FIRST_USER_ADDRESS_EXPR,
+        CheckAccount::new().esdt_balance(GOV_TOKEN_ID_EXPR, rust_biguint!(300)),
+    ));
+    world.check_state_step(CheckStateStep::new().put_account(
+        SECOND_USER_ADDRESS_EXPR,
+        CheckAccount::new().esdt_balance(GOV_TOKEN_ID_EXPR, rust_biguint!(1)),
+    ));
+    world.check_state_step(CheckStateStep::new().put_account(
+        THIRD_USER_ADDRESS_EXPR,
+        CheckAccount::new().esdt_balance(GOV_TOKEN_ID_EXPR, rust_biguint!(800)),
+    ));
+    world.check_state_step(CheckStateStep::new().put_account(
+        OWNER_ADDRESS_EXPR,
+        CheckAccount::new().esdt_balance(GOV_TOKEN_ID_EXPR, rust_biguint!(800)),
+    ));
+}
+
+#[test]
+fn test_down_veto_gov_config() {
+    let mut world = setup();
+    let use_module_whitebox =
+        WhiteboxContract::new(USE_MODULE_ADDRESS_EXPR, use_module::contract_obj);
+
+    let mut current_block_nonce = 10;
+
+    let proposal_id = propose(
+        &mut world,
+        &address_expr_to_address(FIRST_USER_ADDRESS_EXPR),
+        500,
+        &address_expr_to_address(USE_MODULE_ADDRESS_EXPR),
+        b"changeQuorum",
+        vec![1_000u64.to_be_bytes().to_vec()],
+    );
+
+    assert_eq!(proposal_id, 1);
+
+    current_block_nonce += VOTING_DELAY_BLOCKS;
+    world.set_state_step(SetStateStep::new().block_nonce(current_block_nonce));
+
+    world.whitebox_call(
+        &use_module_whitebox,
+        ScCallStep::new()
+            .from(FIRST_USER_ADDRESS_EXPR)
+            .esdt_transfer(GOV_TOKEN_ID, 0, rust_biguint!(300)),
```

Comment:
I think it should work without the rust_biguint! we should have direct conversion from small int